molecule/ci-official-stable3/molecule.yml

```diff
@@ -6,8 +6,8 @@ driver:
 lint:
   name: yamllint
 platforms:
-  - name: grsec_build_jessie
-    image: debian:jessie-20190204
+  - name: grsec_build_stretch
+    image: debian:stretch-20190506
```

Comment:
Is it possible to pin hashes? Some local testing has not been successful (was trying with workstation target):
```
diff --git a/molecule/workstation/molecule.yml b/molecule/workstation/molecule.yml
index ecdb02f..c1ce744 100644
--- a/molecule/workstation/molecule.yml
+++ b/molecule/workstation/molecule.yml
@@ -7,7 +7,7 @@ lint:
   name: yamllint
 platforms:
   - name: grsecurity-build-stable3-stretch
-    image: debian:stretch-20190204
+    image: "debian@sha256:75f7d0590b45561bfa443abad0b3e0f86e2811b1fc176f786cd30eb078d1846f"
 provisioner:
   name: ansible
   lint:
```

with error:
```
t(response, json=decode)\n  File \"/home/m/.virtualenvs/kernels/local/lib/python2.7/site-packages/docker/api/client.py\", line 267, in _result\n    self._raise_for_status(response)\n  File \"/home/m/.virtualenvs/kernels/local/lib/python2.7/site-packages/docker/api/client.py\", line 263, in _raise_for_status\n    raise create_api_error_from_http_exception(e)\n  File \"/home/m/.virtualenvs/kernels/local/lib/python2.7/site-packages/docker/errors.py\", line 31, in create_api_error_from_http_exception\n    raise cls(e, response=response, explanation=explanation)\ndocker.errors.APIError: 500 Server Error: Internal Server Error (\"invalid reference format\")\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
```

Reply:
Ah, yes, we'll have to adjust the create logic to accommodate hashes. Will take a look.

Reply:
Yes, we can pin hashes here, will take some tweaks. Have a WIP locally that let's use declare the Stretch/Xenial hashes once, in an adjacent YAML file, and reuse them across all scenarios, so future updates are DRY. Will need to adjust the CI scenarios to use hash pinning, will ping ya when this is ready for another look, @emkll.